### ISSUE TITLE
Notes in horizontal orientation appear to be parsed incorrectly

### ISSUE THREAD

**Issue by james-xli** [philips/supernote-obsidian-plugin#27]
Notes that were created on a Supernote in the "horizontal" orientation appear to be parsed incorrectly by the plugin. Recognized text appears fine, but the image of the note looks corrupted.

I'm attaching a test .note file here (as .zip file so that Github will accept it):
[horizontal.note.zip](https://github.com/philips/supernote-obsidian-plugin/files/15365765/horizontal.note.zip)

Here is the PNG export of this note from the Supernote, to show what it is supposed to look like:
![horizontal_Page_1](https://github.com/philips/supernote-obsidian-plugin/assets/9400858/63d0d009-14cf-4dd9-8731-36e5944a5587)

Here is how it looks in Obsidian:
![image](https://github.com/philips/supernote-obsidian-plugin/assets/9400858/2cea69ef-3ec1-42aa-b261-33d4c6a7075e)

It's interesting that the image is messed up horizontally, but the vertical relationship of the things on the page is maintained. The image also still looks corrupted after using the Obsidian plugin to export to PNG.

I'm guessing this might be an issue in the underlying supernote-typescript library? Maybe there's some flag in the .note file format or something that identifies when a note is supposed to be horizontal...

### Steps to create a new .note file to reproduce this issue:
1. On the Supernote, open the "Create New Note" window
2. Either "Standard" or "Real-Time Recognition" types will reproduce this issue.
3. Under "Templates," select "Horizontal"
4. Create the .note, write in it, and sync to Obsidian vault as normal.

**Comment by philips** [comment]
Yeah that is 100% a library issue. There are zero examples of horizontal notes in the unit tests

**Comment by philips** [comment]
there are tests in the library and I will upgrade it on the next release. closing.
